styles/_use-cases.scss

```
@@ -543,3 +554,349 @@ html.light {
   padding-left: 0;
   transition: all 0.2s ease-in-out;
 }
+
+.page-tokenization {
+  .tokenization-graphic {
+    content: url('../img/backgrounds/tokenization-illustration.svg');
+    width: 100%;
+    height: 100%;
+  }
+
+  .show-md {
+    display: none;
+    @include media-breakpoint-down(md) {
+      display: block;
+    }
+  }
+
+  .hide-md {
+    display: block;
+    @include media-breakpoint-down(md) {
+      display: none;
+    }
+  }
+
+  .tokenization-use-case {
+    font-size: 12px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 20px;
+    padding-bottom: 10px;
+    border-bottom: 1px solid $gray-600;
+  }
+
+  .tokenization-use-case .arrow-button img {
+    width: 15px;
+    height: 15px;
+  }
+
+  .tokenization-stats {
+    width: 100%;
+    height: 250px;
+    border-radius: 8px;
+    background: linear-gradient(88deg, #9A52FF -14.32%, #32E685 45.35%, #19A3FF 100.76%);
+    padding: 4rem 2rem;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    @include media-breakpoint-down(md) {
+      display: block;
+      height: 100%;
+      width: 100%;
+      padding: 0 25%;
+    }
+  }
+
+  .stat-container {
+    color: $black;
+    text-align: center;
+    border-right: 2px solid black;
+
+    @include media-breakpoint-down(md) {
+      border-right: none;
+      padding-bottom: 3rem;
+      padding-top: 2rem;
+      border-bottom: 2px solid black;
+    }
+    
+  }
+
+  .stat-container:last-child {
+    border: none;
+  }
+
+  .stat-container .stat {
+    font-size: 3rem;
+    font-weight: 300;
+  }
+
+  .stat-container p {
+    font-weight: 400;
+  }
+
+  /* Work-around for border gradient and radius */
+  .security-card {
+    position: relative;
+    border-radius: 0.5rem;
+    background-color: transparent;
+    white-space: normal;
+    box-sizing: border-box;
+  }
+
+  .security-card::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    border-radius: inherit;
+    padding: 1px;
+    background: linear-gradient(90deg, #d91aff 26.41%, #1aa4ff 100.32%);
+    mask: 
+      linear-gradient(#fff 0 0) content-box, 
+      linear-gradient(#fff 0 0);
+    mask-composite: exclude;
+    -webkit-mask-composite: xor;
+  }
```

This effect succeeds at creating a linear-gradient border around the card, but at the cost of making it impossible to click or select the text inside the cards. That can be fixed by adding `z-index`:

```suggestion
  .security-card::before {
    content: '';
    position: absolute;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
    border-radius: inherit;
    padding: 1px;
    background: linear-gradient(90deg, #d91aff 26.41%, #1aa4ff 100.32%);
    mask: 
      linear-gradient(#fff 0 0) content-box, 
      linear-gradient(#fff 0 0);
    mask-composite: exclude;
    -webkit-mask-composite: xor;
    z-index: -1;
  }
```